.flowconfig

```diff
@@ -16,6 +16,8 @@ module.system=node
 suppress_type=$FlowIssue
 suppress_comment= \\(.\\|\n\\)*\\flow:disable
 strip_root=false
+esproposal.class_static_fields=warn
```

Comment:
We should be able to get rid of many of the flow:disable comments. Also valid is `=ignore`.